### PR TITLE
#44 updated web site docs.

### PR DESCRIPTION
**Pull request by ja6a**
Updating the docs for the network and environment resources.

### CODE REVIEW

File: website/docs/r/environment.html.md [hunk 1]
```diff
@@ -3,48 +3,45 @@ layout: "skytap"
...
 
 ```hcl
 # Create a new environment
 resource "skytap_environment" "environment" {
- 	template_id = "123456"
- 	name = "Terraform Example"
- 	description = "Skytab terraform provider example environment."
+  template_id = "123456"
+  name = "Terraform Example"
+  description = "Skytab terraform provider example environment."
```
Comment (lokeuei) [inline website/docs/r/environment.html.md]: Should say Skytap instead of Skytab

File: website/docs/r/environment.html.md [hunk 2]
```diff
@@ -3,48 +3,45 @@ layout: "skytap"
...
+  Provides a Skytap Environment resource.
 ---
 
 # skytap\_environment
 
-Provides a Skytap environment resource. An environment consists of one or more virtual machines, networks, 
+Provides a Skytap Environment resource. An environment consists of one or more virtual machines, networks, 
 and associated settings and metadata. Unlike a template, an environment can be run and have most of its settings 
-modified.
+modified. When an environment is created then its VM's will all be run.
```
Comment (lokeuei) [inline website/docs/r/environment.html.md]: When an environment is created, all of its VMs will be run.

File: website/docs/r/network.html.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `environment_id` - (Required, Force New) ID of the environment you want to attach the network to. If updating with a new one then the network will be recreated.
+* `name` - (Required) User-defined name of the network. Limited to 255 characters. UTF-8 character type.
+* `domain` - (Optional) UDomain name for the Skytap network. Limited to 64 characters.
```
Comment (lokeuei) [inline website/docs/r/network.html.md]: is `UDomain` correct?

File: website/docs/r/network.html.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* `tunnelable` - (Optional) If true, this network can be connected to other networks.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id`: The ID of the network.
+* `environment_id`: ID of the environment the network belongs to. 
+* `name`: User-defined name of the network.
+* `domain`: UDomain name for the Skytap network.
```
Comment (lokeuei) [inline website/docs/r/network.html.md]: same thing here. `UDomain`

File: website/docs/r/environment.html.md (hunk 1 above)
Comment (ja6a) [reply website/docs/r/environment.html.md]: done

File: website/docs/r/environment.html.md (hunk 2 above)
Comment (ja6a) [reply website/docs/r/environment.html.md]: done

File: website/docs/r/network.html.md (hunk 4 above)
Comment (ja6a) [reply website/docs/r/network.html.md]: no! changed to domain

**Pull request merged**